src/Frontend/Components/HeaderComponents/NavBar.js

```diff
@@ -6,8 +6,8 @@ import { useCart } from "../../Context/CartContext.js";
 import { useWishlist } from "../../Context/WishlistContext.js";
```

Comment:
All the contexts can be exported from `index.js` file inside the ` Context` folder. this way the imports will be much cleaner and organized